src/common/HTTPResponse.ts

```diff
@@ -1,29 +1,37 @@
 /**
  * Copyright 2020 Google Inc. All rights reserved.
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
  *
  *     http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
  */
 import { CDPSession } from './Connection';
 import { Frame } from './FrameManager';
 import { HTTPRequest } from './HTTPRequest';
 import { SecurityDetails } from './SecurityDetails';
 import Protocol from '../protocol';
 
-interface RemoteAddress {
+/**
+ * @public
+ */
+export interface RemoteAddress {
   ip: string;
   port: number;
 }
 
+/**
+ * The HTTPResponse class represents responses which are received by page.
```

Comment:
Maybe we could link to the page class here? `responses which are received by instances of the {@link Page} class`

Reply:
Yes and I like the minor tweak of the grammar.